src/Client.php

```diff
@@ -362,13 +362,22 @@ public function disconnect() {
 
     /**
      * Get a folder instance by a folder name
-     * @param $folder_name
+     * @param string $folder_name
+     * @param string|bool $delimiter
      *
      * @return mixed
      * @throws ConnectionFailedException
      * @throws FolderFetchingException
      */
-    public function getFolder($folder_name) {
+    public function getFolder($folder_name, $delimiter = null) {
+        // Set delimiter to false to force selection via getFolderByName (maybe useful for uncommon folder names)
+        if ($delimiter !== false) {
+            $delimiter = (is_null($delimiter)) ? $this->delimiter ?? '/' : $delimiter;
```

Comment:
Hi @DasTobbel ,
thanks for your pull request. A few problems though:
- "??" isn't available in php < 7.0. This library supports php >=5.5.9
- Default delimiter can be accessed via `ClientManager::get('options.delimiter', "/")`

Best regards,